### PR TITLE
Update version number and changelog for v1.13.2-aws release

### PR DESCRIPTION
**Pull request by arunkarthik-akkart**
*Description of changes:*

Update version number and changelog for v1.13.2-aws release

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: RELEASENOTES.md [hunk 1]
```diff
@@ -12,6 +12,26 @@ have unified the code into a single branch, and made the AWS-specific parts a
...
+
+The 1.13.x release series supports
+[NCCL 2.23.4-1](https://github.com/NVIDIA/nccl/releases/tag/v2.23.4-1)
+while maintaining backward compatibility with older NCCL versions
+([NCCL v2.17.1](https://github.com/NVIDIA/nccl/releases/tag/v2.17.1-1) and later)
+
+Bug Fixes:
+  - Resolved issues in the NCCL v3 tuner API to improve stability and ensure correct 
+  behavior across supported platforms and collectives.
+
```
Comment (bwbarrett) [inline RELEASENOTES.md:34]: Add something like:

 - Do not use eager protocol when RDMA writes are in flight.  This improves performance on small AllReduce collectives.

File: RELEASENOTES.md [hunk 2]
```diff
@@ -12,6 +12,26 @@ have unified the code into a single branch, and made the AWS-specific parts a
...
+[the latest-available EFA Installer](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/efa-verify.html)
+for performance improvements and bug fixes.
+
+The 1.13.x release series supports
+[NCCL 2.23.4-1](https://github.com/NVIDIA/nccl/releases/tag/v2.23.4-1)
+while maintaining backward compatibility with older NCCL versions
+([NCCL v2.17.1](https://github.com/NVIDIA/nccl/releases/tag/v2.17.1-1) and later)
+
+Bug Fixes:
+  - Resolved issues in the NCCL v3 tuner API to improve stability and ensure correct
```
Comment (bwbarrett) [inline RELEASENOTES.md:32]: where do things get better?  what's the customer-visible change?

File: RELEASENOTES.md (hunk 1 above)
Comment (rauteric) [inline RELEASENOTES.md:34]: Mention dma-buf turned off.

Comment (arunkarthik-akkart) [reply RELEASENOTES.md:40]: Done

Comment (arunkarthik-akkart) [reply RELEASENOTES.md:40]: Done

File: RELEASENOTES.md (hunk 2 above)
Comment (arunkarthik-akkart) [reply RELEASENOTES.md]: Updated the changelog to provide a more detailed update on the tuner fixes, including better algorithm selection and support for additional bitmask configurations.

**Pull request closed without merging**